meson.build

```diff
@@ -72,12 +78,13 @@ if distro_install
     pkg.generate(lib_inih,
         name : 'inih',
         description : 'simple .INI file parser',
+        extra_cflags : extra_args,
```

Comment:
By only adding that value on MSVC, not on Linux, you can reduce unneeded arguments included in pkg-config files on Linux.